direct/functionals/__init__.py

```diff
@@ -1,9 +1,12 @@
-# coding=utf-8
 # Copyright (c) DIRECT Contributors
 
+"""direct.nn.functionals module."""
+
 from direct.functionals.challenges import *
 from direct.functionals.grad import *
+from direct.functionals.hfen import *
 from direct.functionals.nmae import NMAELoss
 from direct.functionals.nmse import *
 from direct.functionals.psnr import *
+from direct.functionals.snr import *
 from direct.functionals.ssim import *
```

Comment:
Typically you would define an `__all__` if you do it this way.

Reply:
Done!